### ISSUE TITLE
Implement training step

### ISSUE THREAD

**Issue by SMERCA** [SMERCA/TFM_yolo#16]
To implement the training step: 
- [ ] The data must be in the correct format
- [x] the forward of the network must be computed
- [ ] The loss function must be implemented
